tools/run-dev.py

```diff
@@ -77,6 +78,13 @@ def patched_finish(self):
 manage_args = ['--settings=%s' % (settings_module,)]
 os.environ['DJANGO_SETTINGS_MODULE'] = settings_module
 
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
+import scripts.lib.setup_path_on_import
+from django.conf import settings
+
+if not settings.REALMS_HAVE_SUBDOMAINS and not options.test:
+    os.environ["EXTERNAL_HOST"] = "localhost:9991"
+
```

Comment:
Would this be better implemented through just changing zproject/dev_settings.py?  Importing Django here would add 0.7s or so to the startup time for `run-dev.py`.